docs/howto/writing-an-application-from-scratch/index.adoc

```diff
@@ -0,0 +1,99 @@
+
+[id='writing-an-application-from-scratch_{context}']
+= Creating an application from scratch
+
+Creating a simple WildFly Swarm{ndash}based application with a REST endpoint from scratch.
+
+[discrete]
+== Prerequisites
+
+* JDK 8 or newer installed
+* Maven 3.3.x or newer installed
+
+[discrete]
+== Procedure
+
+. Create a directory for the application:
++
+--
+[source,bash,options="nowrap"]
+----
+$ mkdir myApp
+$ cd myApp
+----
+
+We recommend you start tracking the directory contents with Git.
+--
+
+. In the directory, create a `pom.xml` file with the following content.
++
+--
+// Product uses "bom-all" instead of "bom"
```

Comment:
It's the exact opposite -- community uses `bom-all` while product uses `bom`.

Reply:
OK, I'll use `bom` for simplicity then.